src/options.c

```diff
@@ -51,7 +51,7 @@ typedef struct mi_option_desc_s {
 static mi_option_desc_t options[_mi_option_last] =
 {
   // stable options
-  { MI_DEBUG, UNINIT, MI_OPTION(show_errors) },
+  { MI_DEBUG | MI_SHOW_ERRORS, UNINIT, MI_OPTION(show_errors) },
```

Comment:
This condition is still fine if `MI_SHOW_ERRORS` is not defined (as will get value `0` as by the C spec)

Reply:
This may trigger a warning. Are we ok with that?

Reply:
Ah I see. I guess that is  because we use it directly as a value :-(  
Uh, perhaps add an `else` in the `cmake` file to always define `MI_SHOW_ERRORS` as either 1 or 0?

Reply:
Or maybe have some autogenerated part of mimalloc-types.h? This way cmake will take care about defining stuff to 0 for us.

Reply:
For now I rather keep it as is.